cluster/images/conformance/conformance-e2e.yaml

```diff
@@ -59,7 +59,13 @@ spec:
     imagePullPolicy: IfNotPresent
     env:
     - name: E2E_FOCUS
-      value: "Pods should be submitted and removed"
+      value: "\\[Conformance\\]"
+    - name: E2E_SKIP
+      value: "Alpha|\\[(Disruptive|Feature:[^\\]]+|Flaky)\\]"
```

Comment:
... is this skip list still necessary? I thought we had stripped out all Conformance tests that had this 

https://github.com/kubernetes/test-infra/issues/9697#issuecomment-427067895